webpack.config.js

```diff
@@ -45,7 +45,7 @@ const config = {
 							loader: 'sass-loader',
 							query: {
 								includePaths: [ 'editor/assets/stylesheets' ],
-								data: '@import "variables";',
+								data: '@import "variables"; @import "mixins";',
```

Comment:
We still have inconsistency between usage of underscore on the `variables.scss` and `_mixins.scss` files themselves.